bedrock/base/templates/includes/protocol/footer/footer.html

```diff
@@ -6,7 +6,7 @@
 
 {% set utm_params = 'utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=footer' %}
 
-{% if switch('m24-website-refresh', ['en']) %}
+{% if switch('m24-website-refresh') and ftl_file_is_active('footer-refresh') %}
```

Comment:
Issue: (blocking) We should check for both nav and footer here as well, since the CSS files won't be what we need if only the footer is translated.